config/application.rb

```diff
@@ -12,8 +12,11 @@ class Application < Rails::Application
     # Application configuration should go into files in config/initializers
     # -- all .rb files in that directory are automatically loaded.
 
+    config.load_defaults "6.0"
```

Comment:
It looks like a bunch of the test failures are because this sets `cache_versioning` to true, which means that any calls to `cache_key` will no longer include `updated_at`. So caches like these will never be expired:
https://github.com/otwcode/otwarchive/blob/59fa3bb71e4376dd1f61a521e338681f7d6a0716/app/helpers/application_helper.rb#L118
https://github.com/otwcode/otwarchive/blob/59fa3bb71e4376dd1f61a521e338681f7d6a0716/app/views/chapters/_chapter.html.erb#L20
https://github.com/otwcode/otwarchive/blob/59fa3bb71e4376dd1f61a521e338681f7d6a0716/app/controllers/tags_controller.rb#L81
I think you can either disable `cache_versioning`, replace all calls to `cache_key` with `cache_key_with_version`, or start trying to pass a separate `version` parameter to all of the caches.

Reply:
Yeah, I missed that one because it was a 5.2 change. I'd say it makes the most sense to set the config value since we relied on the old behavior in a lot of places.